lcm/lcm_udpm.c

```diff
@@ -264,10 +264,11 @@ static int _recv_message_fragment(lcm_udpm_t *lcm, lcm_buf_t *lcmb, uint32_t sz)
 
     // if this is the first packet, set some values
     char *channel = NULL;
+    int channel_sz = 0;
     if (hdr->fragment_no == 0) {
         channel = (char *) (hdr + 1);
         char *channel = (char *) (hdr + 1);
```

Comment:
I know this wasn't introduced in this PR but if you don't mind, would you remove this line?  It would make reading this part of the code much easier.
```suggestion
```